doc/release-notes.md

```diff
@@ -292,15 +292,18 @@ Wallet
   changed from `-32601` (method not found) to `-18` (wallet not found).
   (#20101)
 
-### Default Wallet
-
-Bitcoin Core will no longer create an unnamed `""` wallet by default when no
-wallet is specified on the command line or in the configuration files. For
-backwards compatibility, if an unnamed `""` wallet already exists and would
-have been loaded previously, then it will still be loaded. Users without an
-unnamed `""` wallet and without any other wallets to be loaded on startup will
-be prompted to either choose a wallet to load, or to create a new wallet.
-(#15454)
+### Automatic wallet creation removed
+
+Bitcoin Core will no longer automatically create new wallets on startup. It will
+load existing wallets specified by `-wallet` options on the command line,
+wallets listed in `bitcoin.conf` or `settings.json` files, and a top-level
+unnamed ("") wallet. But if specified wallets don't exist, Bitcoin Core will log
```

Comment:
bc2b0d99a5f54b941f8c663385f42cb9da20b94b

I've read and then re-read this sentence _"It will load existing wallets specified by `-wallet` options on the command line, wallets listed in `bitcoin.conf` or `settings.json` files, and a top-level unnamed ("") wallet."_

It sounds to me like an existing top-level unnamed ("") wallet is loaded unconditionally. That is not the case, right?

Reply:
> [bc2b0d9](https://github.com/bitcoin/bitcoin/commit/bc2b0d99a5f54b941f8c663385f42cb9da20b94b)
> 
> I've read and then re-read this sentence _"It will load existing wallets specified by `-wallet` options on the command line, wallets listed in `bitcoin.conf` or `settings.json` files, and a top-level unnamed ("") wallet."_
> 
> It sounds to me like an existing top-level unnamed ("") wallet is loaded unconditionally. That is not the case, right?

I am happy to change this paragraph (and please give your suggestion), but I think it is fine. I can't think of any normal sequence of operations that would result in a "" wallet not being loaded and this paragraph isn't trying to translate every condition in wallet loading code to prose, just mention wallets that can be loaded at startup.

Reply:
Suggesting _"It will load existing wallets specified by `-wallet` options on the command line, wallets listed in `bitcoin.conf` or `settings.json` files."_ but not having a strong opinion about that. Feel free to ignore this comment.

Reply:
> Suggesting _"It will load existing wallets specified by `-wallet` options on the command line, wallets listed in `bitcoin.conf` or `settings.json` files."_ but not having a strong opinion about that. Feel free to ignore this comment.

That text seems to match current text exactly, so the suggestion is to stop mentioning the top-level wallet? That seems ok because it's not essential information, but it is information that was in the previous "Default wallet" version of the paragraph, so I'd be reluctant to lose it. It's possible for a top level wallet to not be loaded, but you have to go out of your way to do that, and I don't this release note implies that it's not possible.